pkg/agent/ipassigner/ip_assigner_linux.go

```diff
@@ -31,25 +32,188 @@ import (
 	"antrea.io/antrea/pkg/agent/util/arping"
 	"antrea.io/antrea/pkg/agent/util/ndp"
 	"antrea.io/antrea/pkg/agent/util/sysctl"
+	crdv1b1 "antrea.io/antrea/pkg/apis/crd/v1beta1"
 )
 
-// ipAssigner creates a dummy device and assigns IPs to it.
+// VLAN interfaces created by antrea-agent will be named with the prefix.
+// For example, when VLAN ID is 10, the name will be antrea-ext.10.
+// It can be used to determine whether it's safe to delete an interface when it's no longer used.
+const vlanInterfacePrefix = "antrea-ext."
+
+// assignee is the unit that IPs are assigned to. All IPs from the same subnet share an assignee.
```

Comment:
I don't know if the distinction is meaningful here, but I think there is one assignee per VLAN ID, not per "subnet". Maybe the term "subnet" is a bit misleading here.

Reply:
fixed